Cargo.toml

```diff
@@ -11,7 +11,7 @@ repository = "https://github.com/PistonDevelopers/gfx_text.git"
 documentation = "http://docs.piston.rs/gfx_text/gfx_text/"
 
 [dependencies]
-gfx = "^0.5.2"
+gfx = "^0.6.0"
```

Comment:
I'm confused to see all these `^`, `~`, and `<=` (not only here) in versions. Is this better than `0.6.*`?